notebooks/contributions_quantification.ipynb

```diff
@@ -6,7 +6,7 @@
       "name": "contributions_quantification.ipynb",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    record_ids = {}
</pre><p>add comment here: # select the top data set from each of the top three authors</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Update: Thank you for pointing this out, I have updated the notebook.